Documentation/client.md

```diff
@@ -79,35 +95,39 @@ If you don't yet have a Rook cluster running, refer to our [Quickstart Guides](R
 2. Create an object storage user
 
    ```bash
-   rook object user create rook-user "A rook rgw User"
+   rook object user create rook-user "my object store user"
    ```
 
-3. Get the connection information for accessing object storage
+### Consume the Object Storage
+Use an S3 compatible client to create a bucket in the object store. If you are running in Kubernetes,
+the s3cmd tool is included in the [Rook toolbox](toolbox.md) pod.
```

Comment:
`apk add --no-cache s3cmd` failed for me. But my motivation for removing it was more about keeping the client container pure. If we have an add-on tool, the toolbox just feels like the place to put it.